SECURITY.md

```diff
@@ -363,19 +397,24 @@ CrashOverride will vouch for the "Seven" distribution joining the distribution l
       e-mail updates. This e-mail address will be shared with the Envoy community.
 ```
 
+### Review of membership criteria
+
+In all cases, members of the distribution list will be reviewed on a yearly basis by the maintainers
+and security team to ensure they still qualify for inclusion on the list.
+
 ### Members
 
-| E-mail                                                | Organization  |
-|-------------------------------------------------------|:-------------:|
-| envoy-security-team@aspenmesh.io                      | Aspen Mesh    |
-| aws-app-mesh-security@amazon.com                      | AWS           |
-| security@cilium.io                                    | Cilium        |
-| vulnerabilityreports@cloudfoundry.org                 | Cloud Foundry |
-| secalert@datawire.io                                  | Datawire      |
-| google-internal-envoy-security@google.com             | Google        |
-| argoprod@us.ibm.com                                   | IBM           |
-| istio-security-vulnerability-reports@googlegroups.com | Istio         |
-| secalert@redhat.com                                   | Red Hat       |
-| envoy-security@solo.io                                | solo.io       |
-| envoy-security@tetrate.io                             | Tetrate       |
-| security@vmware.com                                   | VMware        |
+| E-mail                                                | Organization  | Last Review |
```

Comment:
suggestion: add an "end-user" column so that people now how full the 10 slots mentioned above are. And so that it is clear what end users are on the list.